### PR TITLE
fix: add documentation for independent jobs

### PR DESCRIPTION
**Pull request by jithine**
Related to https://github.com/screwdriver-cd/screwdriver/issues/806

### CODE REVIEW

File: docs/user-guide/configuration/workflow.md [hunk 1]
```diff
@@ -142,5 +144,11 @@ jobs:
             - echo: echo hi
 ```
 
+## Re-running a job's build
```
Comment (tkyi) [inline docs/user-guide/configuration/workflow.md]: Not sure this belongs in the configuration section of the documentation.
Maybe move it to https://docs.screwdriver.cd/user-guide/FAQ or just have `### Re-running a job's build` as a section under `## Detached Jobs and Pipelines`

File: docs/user-guide/configuration/workflow.md [hunk 2]
```diff
@@ -142,5 +144,11 @@ jobs:
...
+To re-run a job's build from a past event, do the following steps.
+
+1. Click on desired event from event list, which loads the detailed event graph.
+2. Click on the the job icon from the graph and click "Start pipeline from here"
```
Comment (minzcmu) [inline docs/user-guide/configuration/workflow.md]: Can we add a line here saying that you can see this only after you log in?

**Review by minzcmu — CHANGES_REQUESTED** [review]

**Review by tkyi — APPROVED** [review]
LGTM 👍

**Review by minzcmu — APPROVED** [review]
👍 good to go

**Pull request merged**